components/brave_ads/browser/ads_service_impl_unittest.cc

```diff
@@ -238,6 +238,8 @@ class MockRewardsService : public RewardsService {
                void(brave_rewards::GetWalletPassphraseCallback));
 
   MOCK_METHOD1(SetAdsEnabled, void(const bool is_enabled));
+  MOCK_METHOD0(GetExternalWalletProviders, std::vector<std::string>());
```

Comment:
Any ideas why `MockRewardsService` is declared in `ads_service_impl_unittest.cc` as this mock has nothing to do with `brave_ads`. Also `AdsServiceTest` is not used.

Reply:
I don't think its used as well, I'll remove `AdsServiceTest`

Reply:
removed

Reply:
@jumde Could you please raise an issue to move this file to rewards, thanks